submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -6,42 +6,78 @@ SHOW DATABASES;
  id INT NOT NULL AUTO_INCREMENT,
- fullName VARCHAR(50) NOT NULL,
+ full_name VARCHAR(100) NOT NULL,
  sex VARCHAR(6) NOT NULL,
- age VARCHAR(4),
+ age VARCHAR(4) ,
  PRIMARY KEY(id)
 );
-SELECT * FROM registeredPassengers; 
-
-
+SELECT * FROM registered_passengers;
+select COUNT(*) from registered_passengers;
+DROP table registered_passengers;
 -- Creating the Passenger trip details table
-CREATE TABLE tripDetails (
+CREATE TABLE trip_details (
  id INT NOT NULL AUTO_INCREMENT,
- passengerId INT NOT NULL,
- passengerClass INT NOT NULL,
- passengerTicketNo VARCHAR(20) NOT NULL,
- tripFare DECIMAL(19,4) NOT NULL,
+ passenger_id INT NOT NULL,
+ passenger_class INT NOT NULL ,
+ passenger_ticket_no VARCHAR(50) NOT NULL,
+ trip_fare DECIMAL(19,4) NOT NULL,
  cabin VARCHAR(20),
- numberOfParentsChildren INT NOT NULL DEFAULT  0,
- numberOfSiblingsSpouse INT NOT NULL DEFAULT 0,
- embarkationPoint VARCHAR(255) NOT NULL,
+ number_of_parents_Children INT NOT NULL,
+ number_0f_siblings_spouse    INT NOT NULL,
+ embarkation_point VARCHAR(10) NOT NULL,
  PRIMARY KEY(id),
- FOREIGN KEY(passengerId) REFERENCES registeredPassengers(id)
+ FOREIGN KEY(passenger_id) REFERENCES registered_passengers(id)
 );
-
-SELECT * FROM tripDetails;
+SELECT * FROM trip_details;
+DROP table trip_details;
+select COUNT(*) from trip_details;
 
 -- Creating Accident case table
-CREATE TABLE accidentCases(
+CREATE TABLE accident_cases(
  id int not null auto_increment,
- passengerId int not null,
- accidentStatus tinyint not null,
+ passenger_id int not null,
+ accident_status tinyint not null,
  primary key(id),
- foreign key(passengerId) references registeredPassengers(id)
+ foreign key(passenger_id) references registered_passengers(id)
 );
+Select * from accident_cases;
+select COUNT(*) from accident_cases;
+drop tables accident_cases;
+-- Task 3 Transportation Data Query.
+
+# 1. Total number of passengers who survived
+	select count(*) from accident_cases where accident_status > 0;
+	-- Answers => 342
+
+# 2. total number of unsurvived passengers
+  SELECT COUNT(*) FROM accident_cases WHERE accident_status = 0;
+  -- Answers => 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+	SELECT full_name, sex from registered_passengers
+	inner join trip_details on 
+    registered_passengers.id = trip_details.passenger_id 
+    where (age  < '27') and (embarkation_point = 
+    ( "Q%"| "C%"));
+    
+# 4. Number of passengers who embarked at Southampton and survived
+	SELECT count(*) from accident_cases left join trip_details on 
+    accident_cases.passenger_id = trip_details.passenger_id where 
+    accident_status > 0 and embarkation_point = 'S';
+    -- Answers => 218;
+
+    
+-- total number of passengers who paid a fare greater than $100 and 
+-- above the age of 35 with siblings or spouses on board
+SELECT COUNT(*) FROM registered_passengers 
+inner join trip_details on registered_passengers.id = trip_details.id
```

Comment:
This should be `trip_details.passenger_id`. Though both will give you the same result but, that's not the right thing to do

Reply:
i will change it now, thanks